apps/condo/domains/notification/access/DisconnectUserFromDeviceService.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * Generated by `createservice notification.DisconnectUserFromDeviceService --type mutations`
+ */
+
+/**
+ * Manages DisconnectUserFromDeviceService access rights.
+ * Service can be accessed by both unauthorized and authorized users.
+ * Only deviceId and pushTransport fields are required to disconnect any user from known device successfully.
+ * @param data
+ * @returns {Promise<boolean>}
+ */
+async function canDisconnectUserFromDevice ({ args: { data } }) {
+    if (!data.deviceId || !data.pushTransport) return false
```

Comment:
Why input is validated here?
This utility is for access control checks.
In declaration of custom type `DisconnectUserFromDeviceInput` these fields are marked as required and Keystone should handle it.

Reply:
fixed